modules/base/tool.go

```diff
@@ -206,11 +206,26 @@ func HashEmail(email string) string {
 
 // AvatarLink returns avatar link by given email.
 func AvatarLink(email string) string {
```

Comment:
Just to save one line, you can also declare `url` variable in this line `(url string) { `